### DISCUSSION TITLE
The direction of rainfall seepage is wrong

### DISCUSSION THREAD

**Discussion by LongTIme1111** [Anura3D/Anura3D_OpenSource#214] [Q&A]
Hi
I am simulating the pore water pressure variation of the slope under rainfall, but I find that the matrix suction decreases from the bottom instead of the surface first, and I set the soil surface as follows
![853a520e14494fb3bcbdc5240ee3dad](https://github.com/Anura3D/Anura3D_OpenSource/assets/162860598/eef16e59-55f0-4651-8969-2622d7307dd5)
![964f148f3ae172aeced883e52622e01](https://github.com/Anura3D/Anura3D_OpenSource/assets/162860598/ea15240d-6be1-410e-8d06-d0b054d44820)
The matrix suction varies as follows
![75bbd465ee20835a711227edaa01fc3](https://github.com/Anura3D/Anura3D_OpenSource/assets/162860598/e4e66e0f-38f9-48a8-bfe4-767a181e4755)
![ad93401213f94ca59e666b617be82d8](https://github.com/Anura3D/Anura3D_OpenSource/assets/162860598/c8e6c270-ed1a-49b2-a6ca-905a4d7c7ddb)
![86045f5b587f53ebb78d2dcf5637b06](https://github.com/Anura3D/Anura3D_OpenSource/assets/162860598/44fe434c-1487-4e20-a3c4-1c14bb3696de)
![59f20d49b8a835aa2d584008fd0a1e3](https://github.com/Anura3D/Anura3D_OpenSource/assets/162860598/d0d8eb5b-ec03-4c5b-8290-d3457016ace5)
![44bd281ee20640eee50ccbd4a216b5c](https://github.com/Anura3D/Anura3D_OpenSource/assets/162860598/174c8450-5682-4d70-a243-a878ee1357b6)
Hope to get any reply, thanks！！！！

**Comment by elbarrendero** [comment]
Good night, how are you defining the x,y min and x,y max, this area defines your direction if later on you put negative infiltration it will go the other way, think about the x,y min,max as a vector.
